src/ServiceProvider.php

```diff
@@ -92,4 +94,28 @@ protected function registerPermissions()
             })->group('Runway');
         }
     }
+
+    protected function bootGraphQl()
+    {
+        Runway::allResources()
+            ->filter
+            ->graphqlEnabled()
+            ->each(function (Resource $resource) {
+                $this->app->bind("runway.graphql.queries.{$resource->handle()}.index", function () use ($resource) {
+                    return new \DoubleThreeDigital\Runway\GraphQL\ResourceIndexQuery($resource);
+                });
+
+                $this->app->bind("runway.graphql.queries.{$resource->handle()}.show", function () use ($resource) {
+                    return new \DoubleThreeDigital\Runway\GraphQL\ResourceShowQuery($resource);
+                });
+
+                $this->app->bind("runway.graphql.types.{$resource->handle()}", function () use ($resource) {
+                    return new \DoubleThreeDigital\Runway\GraphQL\ResourceType($resource);
+                });
+
+                GraphQL::addType("runway.graphql.types.{$resource->handle()}");
```

Comment:
Maybe these need to happen before Statamic is booted in order for them to show in GraphiQL?